docs/how_to_add_a_publisher.md

```diff
@@ -333,14 +335,140 @@ def title(self) -> Optional[str]:
 
 ### Extracting Attributes with XPath and CSS-Select
 
-Sometimes you have to get information directly from the DOM of the HTML (in most cases the article text).
-To do so we suggest using [**lxml**](https://lxml.de/) and selectors like [`XPath`](https://lxml.de/xpathxslt.html) or [`CSSSelect`](https://lxml.de/cssselect.html).
-You can use both selectors on the `doc` attribute of `Precomputed`.
-If you use a selector in any of your attributes (which will be most likely the case when extracting the actual article text) you should precompute the selector as a class variable using the `XPath` respectively `CSSSelector` classes of lxml.
+When parsing the `ArticleBody`, or in certain scenarios, you need to directly obtain information from the [Document Object Model](https://en.wikipedia.org/wiki/Document_Object_Model) (DOM) of the HTML/XML.
+The DOM serves as an interface representing the underlying HTML or XML file as a tree structure, where each element (tag) of the file functions as a node in the tree.
+To select or search respectively for the information you need you can access these nodes using selectors like CSS-Select or XPath.
+Fundus relies on the Python package `lxml` and its selector implementation.
```

Comment:
Maybe you could add a sentence along the lines of " if you are new to this, Scimming the documentation of lxml is advised"

Reply:
Actually, i wouldn't suggest going to the `lxml` documentation first. Most of the time it's an information wall and pretty overwhelming if you're new to the topic. I think our entry examples are a good starting points, IMO actually a better starting point than skimming the `lxml` doc

Reply:
That's a very good point, I agree.